dailymotion-player-objc/DMPlayerViewController.m

```diff
@@ -43,6 +45,7 @@ - (void)setup {
   _duration = NAN;
   _seeking = false;
   _error = nil;
+  _started = false;
```

Comment:
`false` should be `NO` (line 192 uses the correct `YES`) and actually no need to initialise the values at all, ObjC defaults to the falsey values.